ts/session/types/RawMessage.ts

```diff
@@ -10,3 +10,12 @@ export interface RawMessage {
   ttl: number;
   encryption: EncryptionType;
 }
+
+export interface BareRawMessage {
```

Comment:
What would this interface be used for?

Reply:
RawMessage when plainTextBuffer is yet to be rebuilt

Reply:
Hmm would there be a better name 🤔

Reply:
Fixed